tube/etl/plugins/mapping.py

```diff
@@ -1,8 +1,11 @@
 def add_auth_resource_path_mapping(doc_type, types):
     properties = types.get(doc_type).get("properties")
-    if "project_id" in properties:
-        types[doc_type]["properties"]["auth_resource_path"] = {
-            "type": "keyword",
-            "fields": {"analyzed": {"type": "text"}},
-        }
+    types[doc_type]["properties"]["auth_resource_path"] = {
+        "type": "keyword",
+        "fields": {"analyzed": {"type": "text"}},
+    }
+    types[doc_type]["properties"]["project_id"] = {
+        "type": "keyword",
+        "fields": {"analyzed": {"type": "text"}},
+    }
```

Comment:
i think i understand why you removed the `if` statement and are adding `types[doc_type]["properties"]["auth_resource_path"]` all the time.

but is adding `types[doc_type]["properties"]["project_id"]` part of this fix, or is it a separate fix? how is it related to this error?